### PR TITLE
Add initial Helm chart generator script

### PR DESCRIPTION
**Pull request by ArangoGutierrez**

### CODE REVIEW

**Review by visheshtanksale — APPROVED** [review]
LGTM!! 
Can we also add a make target?

**Comment by ArangoGutierrez** [comment]
> LGTM!! Can we also add a make target?

Done

File: hack/package-helm-charts.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
```
Comment (shivamerla) [inline hack/package-helm-charts.sh:1]: Both of these seem redundant, can we just use the make target from the pipeline as well.

Comment (ArangoGutierrez) [reply hack/package-helm-charts.sh:1]: makes sense

Comment (ArangoGutierrez) [reply hack/package-helm-charts.sh]: done

File: Makefile [hunk 2]
```diff
@@ -145,6 +145,11 @@ build-installer: manifests generate kustomize ## Generate a consolidated YAML wi
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default > dist/install.yaml
 
+# Short version of hack/package-helm-charts.sh
+.PHONY: helm-charts
+helm-charts:
```
Comment (shivamerla) [inline Makefile:150]: @ArangoGutierrez lets add all CI changes required for Helm packaging together in one MR. In this case its not clear which image will be used with this target as that will not match with `VERSION`.

Comment (shivamerla) [reply Makefile:150]: sounds good.

**Review by shivamerla — APPROVED** [review]
lgtm

**Pull request merged**